doc/reference/reference_lua/msgpack.rst

```diff
@@ -528,7 +538,7 @@ userdata/cdata, you can use this code:
 .. NOTE::
 
     To achieve the same effect for only one call to ``msgpack.encode()``
-    (i.e. without changing the configuration permanently), you can use
+    (that is without changing the configuration permanently), you can use
     ``msgpack.encode({1, x, y, 2}, {encode_invalid_numbers = true})``.
```

Comment:
This isn't true:

```
tarantool> msgpack.encode({1, 2}, {encode_invalid_numbers = true})
---
- error: 'msgpack.encode: argument 2 must be of type ''struct ibuf'''
...
```

It's possible to do this though:

```Lua
msgpack.new({encode_invalid_numbers = true}).encode({1, 2})
```

Reply:
Thanks. Also found out the the example above with `httpc.curl` doesn't work:
```
tarantool> msgpack.encode(httpc.curl)
---
- error: unsupported Lua type 'userdata'
...

tarantool> msgpack.encode(httpc.curl, {encode_use_tostring=true})
---
- error: 'msgpack.encode: argument 2 must be of type ''struct ibuf'''
...
```

Changed to `msgpack.cfg{encode_use_tostring = true}`.